packages/mdc-list/_mixins.scss

```diff
@@ -113,7 +113,7 @@
 
   .mdc-list-item__meta {
     // stylelint-disable selector-class-pattern
-    &:not(.material-icons) {
+    &:not(.material-icons):not(.material-icons-outlined):not(.material-icons-round):not(.material-icons-two-tone):not(.material-icons-sharp) {
```

Comment:
This increases specificity, we should create a modifier for meta text and meta icon that applies specific styles.

Example: `.mdc-list-item__meta--text`

Reply:
Yes, that's would be great.